### PR TITLE
added centos support

### PR DESCRIPTION
**Pull request by atheiman**
first run at issue #37

### CODE REVIEW

File: recipes/_nginx.rb [hunk 1]
```diff
@@ -1,6 +1,19 @@
+cert = ssl_certificate 'drone' do
```
Comment (atheiman) [inline recipes/_nginx.rb]: Moved so that the cert can be available to the nginx template.

File: templates/default/nginx-default-site.erb [hunk 2]
```diff
@@ -8,8 +8,8 @@ server {
   listen 443 ssl;
   listen [::]:443 ssl;
 
-  ssl_certificate /etc/ssl/certs/drone.pem;
-  ssl_certificate_key /etc/ssl/private/drone.key;
+  ssl_certificate <%= @cert.cert_path %>;
```
Comment (atheiman) [inline templates/default/nginx-default-site.erb]: Dont make assumptions about the location of the cert

**Comment by atheiman** [comment]
@jmccann let me know if you have any thoughts on this pr please

**Comment by jmccann** [comment]
Thanks for the contribution!

**Pull request merged**